.github/workflows/scripts/ci-reverse-lab-scan.rb

```diff
@@ -0,0 +1,23 @@
+# Copyright (c) 2024 SolarWinds, LLC.
+# All rights reserved.
+
+require 'json'
+
+scan_report_path = '../../../reports/report.checks.json'
+if File.exist?(scan_report_path)
+	content = File.read(scan_report_path)
+	parsed_data = JSON.parse(content)
+	assessments = parsed_data['report']['scans']['scan-version']['assessments']
+
+	assessments.each do |key, value|
+		if value['status'] != 'pass'
+			puts "Found issue. Please check https://my.secure.software/."
+			exit(1)
+		end
+	end
+else
+	puts "Missing scanned report."
+	exit(1)
+end
+
+exit(0)
```

Comment:
Could maybe add a confirmation message such as

```suggestion
puts "Scan report complete. No issues"
exit(0)
```